public/js/services.js

```diff
@@ -1,5 +1,5 @@
-angular.module('starter.services', [])
-  .factory("api", function($q, $http) {
+/* @ngInject */
```

Comment:
There's a build task in gulp which run's ngAnnotate, when doing the minification, so this should be good.